.github/workflows/images-legacy.yaml

```diff
@@ -170,7 +170,7 @@ jobs:
       - name: Upload artifact digests
         uses: actions/upload-artifact@ee69f02b3dfdecd58bb31b4d133da38ba6fe3700
         with:
-          name: image-digest ${{ matrix.name }}
```

Comment:
What's the issue with the old version? [The upload-artifact's README](https://github.com/actions/upload-artifact) has examples with whitespaces so I'd expect that to work :confused:

Reply:
You are right, I pushed the wrong fix. It is the file path that is wrong xD

Reply:
Fixed in latest push.